packages/lib-utils/src/components/status/StatusBox.tsx

```diff
@@ -0,0 +1,99 @@
+import { Alert, EmptyState, EmptyStateBody, Spinner, Text, Title } from '@patternfly/react-core';
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+export type LoadError = {
+  message: string;
+  status: number;
+};
+
+type AccessDeniedProps = {
+  message: string | React.ComponentType;
+};
+
+export const AccessDenied: React.FC<AccessDeniedProps> = ({
+  message = 'Insufficient access rights.',
+}) => (
+  <EmptyState>
+    <Title headingLevel="h2">Restricted Access</Title>
+    <EmptyStateBody>
+      <Text>You don&apos;t have access to this section due to cluster policy.</Text>
```

Comment:
If it makes sense, I'd suggest a follow-up change where we use props instead of hard-coded strings like the one above.

Reply:
I'll add this to a follow-up list. We probably have to discuss it a bit. In the original console's implementation, there is a translated string being rendered. Since we don't have translations yet, I put just this string for now.

Reply:
@fhlavac @florkbr Well, there are two aspects to this, first one is localization (for which we currently don't have support in the SDK repo yet), while the second one is reusability (hard-coded strings, regardless if localized or not, make a component less reusable).

Ideally, for every component we move from Console to `lib-utils` in the SDK repo, we should re-evaluate its API and code. I agree that initially we want to have some components ready and good-enough to be used by HAC Core, this is more for a long-term component & API related discussion.